war/src/main/js/templates/pluginSelectionPanel.hbs

```diff
@@ -40,6 +39,20 @@
 					<span class="description">
 						{{{plugin.excerpt}}}
 					</span>
+					{{#hasDependencies plugin.name}}
+					<span class="deps">
+						<div class="btn btn-link toggle-dependency-list {{#ifVisibleDependency ../plugin.name}}active{{/ifVisibleDependency}}" type="button" data-plugin-name="{{../plugin.name}}">
+							{{../../../translations.installWizard_installIncomplete_dependenciesLabel}}<span class="badge">{{dependencyCount ../plugin.name}}</span>
+						</div>
+						<div class="dep-list">
+							{{#eachDependency ../plugin.name}}
+							<span class="dep badge">
+							{{title}}
+							</span>
+							{{/eachDependency}}
+						</div>
+					</deps>
```

Comment:
I have no idea how to debug this beast but `</deps>` does not seem to be a valid closing tag. It should be `</span>`.